bot.py

```diff
@@ -1,545 +1,753 @@
+        if l1 < l2 or l2 < l1:
+            return True
         else:
-            return 'Недостаточно прав для выполнения команды. Вероятно, гостевой аккаунт заблокирован. Купить код приглашения можно тут: https://t.me/routerscan/15931'
-    else:
-        return 'Что-то пошло не так 😮 error: ' + error
-
-
-def apiquery(user_id, bssid='*', essid=None, sensivity=False):
-    """Implements querying single AP by its BSSID/ESSID"""
-    api_key = getPersonalAPIkey(user_id)
-    url = f'{SERVICE_URL}/api/apiquery?key={api_key}&bssid={bssid}'
-    if essid is not None:
-        url += f'&essid={essid}'
-    if sensivity:
-        url += '&sens=true'
-    response = requests.get(url).json()
-
-    reply_markup = None
-    if not response['result']:
-        return getApiErrorDesc(response['error'], user_id), reply_markup
-    if len(response['data']) == 0:
-        if bssid != '*':
-            keyboard = [
-                [
-                    InlineKeyboardButton('Сгенерировать пин-коды WPS',
-                                         callback_data=f'{user_id}/{bssid}')
-                ]
-            ]
-            reply_markup = InlineKeyboardMarkup(keyboard)
-        return 'Нет результатов :(', reply_markup
-    return formataps(tuple(response['data'].values())[0]), reply_markup
-
-
-def apiwps(user_id, bssid):
-    """Implements generating PIN codes for single AP by its BSSID"""
-    api_key = getPersonalAPIkey(user_id)
-    response = requests.get(
-        f'{SERVICE_URL}/api/apiwps?key={api_key}&bssid={bssid}'
-    ).json()
-    if not response['result']:
-        return getApiErrorDesc(response['error'], user_id)
-    if len(response['data']) == 0:
-        return 'Нет результатов :('
-    return formatpins(response['data'][bssid.upper()]['scores'])
-
-
-def parseApDataArgs(args):
-    """Parsing BSSID and ESSID from /pw command arguments"""
-    if re.match(bssid_pattern, args[0]) is not None:
-        bssid = args[0]
-        if len(args) > 1:
-            essid = ' '.join(args[1:])
+            return False
+    except:
+        return False
+
+# bot #
+bot = Bot(token = cfg.token)
+dp = Dispatcher()
+
+# callbacks #
+class WpsCallback(CallbackData, prefix = 'wps'):
+    wps: int
+class LangCallback(CallbackData, prefix = 'lang'):
+    lang: str
+class MaydayCallback(CallbackData, prefix = "md"):
+    turn: int
+
+# logger #
+logform = '%(asctime)s - %(name)s - %(levelname)s - %(message)s'
+logging.basicConfig(format = logform, level = logging.DEBUG)
+log = logging.getLogger(__name__)
+
+# filters #
+MAC = r'(?:[0-9A-Fa-f]{2}[:-]){5}(?:[0-9A-Fa-f]{2})'
+
+# bot body #
+@dp.message(commands = ['start'])
+async def start_mess(message: types.Message):
+    if users.getstatus(message) == Status.AUTH or users.getstatus(message) == Status.MAYDAY:
+        await help_mess(message)
+    elif users.getstatus(message) == Status.UNAUTH:
+        await message.reply(lng.getmess(message, "start"), parse_mode = "Markdown")
```

Comment:
```suggestion
        await message.reply(lng.getmess(message, "start").format(url = SERVICE_URL), parse_mode = "Markdown")
```
SERVICE_URL из словаря в start